src/GameLogic/ItemExtensions.cs

```diff
@@ -146,17 +146,17 @@ public static bool IsJewelry(this Item item)
     /// </returns>
     public static bool IsPhysicalWeapon(this Item item, [NotNullWhen(true)] out float? minimumDmg)
     {
-        minimumDmg = item.Definition?.BasePowerUpAttributes.FirstOrDefault(a => a.TargetAttribute == Stats.MinimumPhysBaseDmg)?.BaseValue;
+        minimumDmg = item.Definition?.BasePowerUpAttributes.FirstOrDefault(a => a.TargetAttribute == Stats.MinimumPhysBaseDmgByWeapon)?.BaseValue;
```

Comment:
This was stopping excellent/ancient bonus from being added.
(check spreadsheet in PR body for before and after values)